### Message type for live configurate the slider

# PID Controller parameters
float64 P_gain
float64 I_gain
float64 D_gain